docker/stan-wasm-server/logic/fileValidation/compilationFiles.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+# 10 MB limit for Stan source files
+MAX_STAN_SRC_FILESIZE = 1024 * 1024 * 10
+VALID_STAN_SRC_FILENAME = 'main.stan'
+VALID_STAN_COMPILATION_OUTPUTS_FILENAMES = ['main.js', 'main.wasm']
+
+def download_filename_is_valid(filename: str):
+    return filename in VALID_STAN_COMPILATION_OUTPUTS_FILENAMES
+
+
+def _stan_src_filename_is_valid(filename: str):
+    return filename == 'main.stan'
```

Comment:
Why are we even accepting the filename as part of the request if we require it to be exactly `main.stan`?

Reply:
I assumed it was for potential future feature support; it was in the current version.

Reply:
I think we should just make it so the Stan file always has a consistent name on disk. If we want to make it so the user can supply a file name which is printed in messages etc, we can do that without actually renaming the file (through an argument called `--filename-in-msg` to stanc3) and then we don't really care _what_ they request, `not a stan file.txt` would be fine

Reply:
(realizing the above isn't actionable)

I think it makes sense to accept a string as a filename in the payload but currently just do nothing at all with it.